one-page.js

```diff
@@ -373,6 +365,36 @@ function inlineAssets(projectPath) {
                 }
             })();
 
+            // Add Snapchat/Google CTA code if needed
+            (function() {
+                switch(config.one_page.cta_option) {
+                case 'snapchat':
+                    console.log("↪️ Adding Snapchat Ad CTA code");
+                    addLibraryFile('snapchat-cta.js');
+                break;
+                case 'google':
+                    console.log("↪️ Adding Google Ad CTA code");
+                    var allowedValues = ["portrait", "landscape", "portrait,landscape"];
+                    if(config.one_page.adOrientation && allowedValues.includes(config.one_page.adOrientation)) {
+                        var orientation = config.one_page.adOrientation;
+                        var size = (orientation == 'portrait') ? 'width=320,height=480' : 'width=480,height=320\n';
```

Comment:
Should ad_orientation logic be outside the google CTA given it's a separate property in the config?

Happy to leave it here for now as I don't know if this is a common meta property outside of Google and we can always pull this out later

Reply:
I was also wondering about this. I almost changed cta_option to be an object based property, but it didn't feel right to have those settings embedded given that orientation and size don't directly relate to a CTA.

Reply:
I would leave it as is for now, we can always change it later